src/backend/admin/admin.plugin.ts

```diff
@@ -0,0 +1,44 @@
+import mongoose from 'mongoose';
+import AdminBroExpress from 'admin-bro-expressjs';
+import AdminBroMongoose from 'admin-bro-mongoose';
+import session from 'express-session';
+import connectMongo from 'connect-mongo';
+
+import config from '../config/admin';
+import getAdminAuthenticationConfig from '../admin/authentication/admin-auth.config';
+
+import { Environment } from '../config/environments';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const AdminBro = require('admin-bro');
```

Comment:
AFAIK all typings are fixed in admin bro so it can be imported normally
```suggestion
import AdminBro from 'admin-bro';
```